content/reference-transaction-format.md

```diff
@@ -1049,6 +1049,7 @@ These codes indicate an error in the local server processing the transaction; it
 | telBAD\_DOMAIN | The transaction specified a domain value (for example, the `Domain` field of an [AccountSet transaction](#accountset)) that cannot be used, probably because it is too long to store in the ledger. |
 | telBAD\_PATH_COUNT | The transaction contains too many paths for the local server to process. |
 | telBAD\_PUBLIC\_KEY | The transaction specified a public key value (for example, as the `MessageKey` field of an [AccountSet transaction](#accountset)) that cannot be used, probably because it is too long. |
+| telCAN\_NOT\_QUEUE | The transaction did not meet the [open ledger cost](concept-transaction-cost.html), but this server did not queue it because it failed one of the server's heuristics for being "likely to succeed". You can try again later or sign and submit a replacement transaction with a higher transaction cost in the `Fee` field. |
```

Comment:
I don't think there are any cases where a higher fee will overcome `telCAN_NOT_QUEUE`. (Any where it would should still return `telINSUF_FEE_P`.)

Here's a complete list of what will return a `telCAN_NOT_QUEUE` as of 0.32.0:
* Attempting to replace a tx sequence already in the queue (but not the first or last) with a tx that affects how the address authorizes accounts
* Attempting to queue a new (later sequence) tx when a tx already in the queue affects how the address authorizes accounts.
* Attempting to queue a new tx or replace a queued tx when all the other queued txs already potentially leave the account with too low of a balance to cover the fee or account reserve.
* The tx would otherwise pass the heuristics to be queued, but the account has more than 10 txs already queued, the tx has too low a `LastLedgerSequence`, or the tx has a `AccountTxnID` or `PreviousTxnID`.
* The tx would otherwise pass heuristics, but the queue is full, and lowest fee tx in the queue is for the same account.

In short, there really isn't much a user can do to the current candidate transaction to overcome `telCAN_NOT_QUEUE`. That's pretty much why it needed a new result code. The only options I can think of are to wait or to try to replace / reprocess some of the earlier transactions in the queue (for example, replace the first queued tx with one that pays the open ledger fee. This will, of course, require a lot more work on the client to track such things.)